rapids-cmake/cuda/set_architectures.cmake

```
@@ -89,10 +94,10 @@ function(rapids_cuda_set_architectures mode)
     list(TRANSFORM CMAKE_CUDA_ARCHITECTURES APPEND "-real")
   endif()
 
-  # CUDA 12.8.0 and later warns when compiling for arch 70. We ignore this warning when compiling
+  # For the CUDA 12.X.0 series we want to silence warnings when compiling for arch 70 when compiling
   # for RAPIDS architectures.
-  if(CMAKE_CUDA_COMPILER_ID STREQUAL "NVIDIA" AND CMAKE_CUDA_COMPILER_VERSION VERSION_GREATER_EQUAL
-                                                  12.8.0)
+  if(CMAKE_CUDA_COMPILER_ID STREQUAL "NVIDIA" AND CMAKE_CUDA_COMPILER_VERSION VERSION_LESS_EQUAL
+                                                  12.9.99)
```

I'm a little confused. I would have expected something more like this:

```suggestion
  if(CMAKE_CUDA_COMPILER_ID STREQUAL "NVIDIA" AND CMAKE_CUDA_COMPILER_VERSION VERSION_GREATER_EQUAL 12.8.0 AND CMAKE_CUDA_COMPILER_VERSION VERSION_LESS
                                                  13.0.0)
```